addons/hr_holidays/models/hr_leave.py

```diff
@@ -128,10 +128,12 @@ def _default_get_request_parameters(self, values):
     user_id = fields.Many2one('res.users', string='User', related='employee_id.user_id', related_sudo=True, compute_sudo=True, store=True, readonly=True)
     manager_id = fields.Many2one('hr.employee', compute='_compute_from_employee_id', store=True, readonly=False)
     # leave type configuration
+    def _default_leave_type(self):
+        return self.env['hr.leave.type'].search([('name', '=', 'Sick Time Off')]).id
```

Comment:
Hello, I don't really know the current state of the PR, and it's not really my module, but just a suggestion as I am passing by:

Be careful when using hard-coded string values in the code, it is often the wrong way to approach a problem. Indeed, as the value is in database, it could be updated by the user, and then your query will no longer work. What's even worse is that when the value is translatable, even if the user doesn't change it, it will only work for English users.

Reply:
PS: is it a good spec functionally? It's really annoying to me on our prod that sick is the default value. I'd personally rather have nothing as default, than something always wrong. (a sensitive default would be bank holidays compensation first, then legal leave if none is available, then extra leave if none is available). In order where we legally should take them. Sick is really an exception.

Reply:
PS2: don't forget to use "draft" PR in github, if you want to avoid notifying people if it's still work in progress. Feel free to reset the status to draft if that is indeed still the case.

Reply:
> PS: is it a good spec functionally? 
I will discuss that with gmf, thanks for the insight.

> PS2: don't forget to use "draft" PR in github, if you want to avoid notifying people if it's still work in progress. Feel free to reset the status to draft if that is indeed still the case.
I wasn't aware that I was notifying people when the PR is not in draft, sorry for that !